sw/device/lib/crypto/impl/ecc/ecdh_p256.c

```diff
@@ -33,8 +33,6 @@ static const otbn_addr_t kOtbnVarEcdhD1 = OTBN_ADDR_T_INIT(p256_ecdh, d1);
 static const uint32_t kOtbnEcdhModeWords = 1;
 static const uint32_t kOtbnEcdhModeKeypairRandom = 0x3f1;
 static const uint32_t kOtbnEcdhModeSharedKey = 0x5ec;
-static const uint32_t kOtbnEcdhModeKeypairFromSeed = 0x29f;
-static const uint32_t kOtbnEcdhModeSharedKeyFromSeed = 0x74b;
```

Comment:
@jadephilipoom pinging just in case you need these in the future

Reply:
Thanks for the heads up! I'll eventually need them for sideloaded keys, but I can recover the values from the assembly file.